pkg/config/ao.go

```diff
@@ -25,9 +25,9 @@ var ocp3URLPatterns = &ServiceURLPatterns{
 var ocp4URLPatterns = &ServiceURLPatterns{
 	ClusterURLPattern:      "https://api.%s.paas.skead.no:6443",
 	ClusterLoginURLPattern: "https://oauth-openshift.apps.%s.paas.skead.no",
-	BooberURLPattern:       "http://boober.aurora.apps.%s.paas.skead.no",
-	UpdateURLPattern:       "http://ao-aurora-tools.%s.paas.skead.no",
-	GoboURLPattern:         "http://gobo.aurora.apps.%s.paas.skead.no",
+	BooberURLPattern:       "https://boober-aup.apps.%s.paas.skead.no",
+	UpdateURLPattern:       "https://ao-aup-tools.apps.%s.paas.skead.no",
```

Comment:
Må det ikke endres kode for deploy av serveren for oppdateringer også, dersom denne skal endres til å være ulik den for ocp3?

Reply:
Godt poeng, den må legges til i listen over availableUpdateClusters.